cg/test/subjects/subjects.api.spec.js

```diff
@@ -146,12 +146,14 @@ describe('Tests of subject giving consent', () => {
         Authorization: `Bearer ${token}`
       }
     });
-
+    // console.log(await res.text());
```

Comment:
Please remove these commented out logs. Including all the below occurrences.

Reply:
k